example/conf/default.yaml

```diff
@@ -1,6 +1,24 @@
 testValue: 123
 mongo:
   db: 'rapptor-example'
+logging:
+  reporters:
+    hipchat:
+      - 
+        auth_token: ''
+        room: ''
+        roomToken: ''
+        debug: true
+        notify: false
+        events:
```

Comment:
I like the enable flag.  I have that on the plugins side, make sense to do on the logging side